admin/client/src/legacy/LeftAndMain.BatchActions.js

```diff
@@ -271,39 +242,49 @@ $.entwine('ss.tree', function($){
     },
 
     onsubmit: function(e) {
-      var self = this, ids = this.getIDs(), tree = this.getTree(), actions = this.getActions();
-      
+      var self = this,
+        ids = this.getIDs(),
+        tree = this.getTree(),
+        actions = this.getActions();
+
       // if no nodes are selected, return with an error
       if(!ids || !ids.length) {
         alert(i18n._t('CMSMAIN.SELECTONEPAGE', 'Please select at least one page'));
         e.preventDefault();
         return false;
       }
-      
+
       // apply callback, which might modify the IDs
-      var type = this.find(':input[name=Action]').val();
+      var actionURL = this.find(':input[name=Action]').val();
+      if (!actionURL) {
+        e.preventDefault();
+        return false;
+      }
+
+      // Validate action
+      var type = actionURL.split(/\//).filter(n => true).pop();
```

Comment:
I'd suggest `.split("/")` for no need for Regex reasons.
What does the `.filter(n => true)` do? To me it isn't doing anything (empty values are retained)